extensions/exchanges/bitfinex/exchange.js

```diff
@@ -330,6 +343,9 @@ module.exports = function container (get, set, clear) {
     },
     
     trade: function (action, opts, cb) {
+      if (!pair) { pair = joinProduct(opts.product_id) }
+      var symbol = 't' + pair
+
```

Comment:
Not a duplicate. The symbol on [trade needs a 't' prefix](https://bitfinex.readme.io/v2/docs/ws-general), and in case the trade function is ever called as the first function, we need to know the pair and set it globally.